testing/test_gl_surface.cc

```diff
@@ -83,10 +85,26 @@ static std::string GetEGLError() {
 
 TestGLSurface::TestGLSurface(SkISize surface_size)
     : surface_size_(surface_size) {
-  display_ = ::eglGetDisplay(EGL_DEFAULT_DISPLAY);
+  PFNEGLGETPLATFORMDISPLAYEXTPROC egl_get_platform_display_EXT =
```

Comment:
In GL and EGL, that the proc address is available doesn't mean that the extension is available as well. Please perform an extensions check for [`EGL_EXT_platform_base`](https://www.khronos.org/registry/EGL/extensions/EXT/EGL_EXT_platform_base.txt) perhaps and also for any Angle specific extensions.

Reply:
Will do